symfony/templates/error-gae-auth.html.twig

```diff
@@ -0,0 +1,8 @@
+{% extends 'base.html.twig' %}
+{% block title %}Error - {% endblock %}
+{% block body %}
+    <div class="alert alert-danger">
+        <i class="fa fa-minus-circle"></i>
+        <strong>Access denied!</strong> <a href="{{ logoutUrl }}">Please try again</a>.
```

Comment:
Sorry, one more minor thing. Can we change this message to the one we previously used in the flash message ("Authentication failed. Please try again.") because if the OAuth process failed for some reason, they would see this message.

And can we change this template name to something else, since "gae" no longer applies? (A very long time ago, we were using an App Engine-specific auth process, instead of the regular Google OAuth workflow that we use now). Maybe name this one `error-auth.html.twig`, and then rename the other two to `error-auth-2fa.html.twig` and `error-auth-groups.html.twig`?